### PR TITLE
feat : my team member

### LINKED ISSUE #2: my team's introduce
- [x] team name
- [x] member
- [x] goal

### PR DESCRIPTION
**Pull request by sians5544**
first simple member introduce  
resolve #2

### CODE REVIEW

File: GROUP.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+
+1.team name : goodzhunhe
+2. member: 조용우,박시안, 허준혁
```
Comment (zhunhe) [inline GROUP.md:4]: Please use space !

**Review by zhunhe — CHANGES_REQUESTED** [review]

File: GROUP.md (hunk 1 above)
Comment (sians5544) [reply GROUP.md:4]: ok...

File: GROUP.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 
+# 우리 조 소개
+
+## team name : goodzhunhe
+
+## member: 조용우,박시안, 허준혁
```
Comment (zhunhe) [inline GROUP.md:6]: Edit from "조용우,박시안" to "조용우, 박시안" !

**Review by zhunhe — CHANGES_REQUESTED** [review]

File: GROUP.md (hunk 2 above)
Comment (sians5544) [reply GROUP.md:6]: sure~~

**Review by zhunhe — APPROVED** [review]

**Pull request merged**